tools/bumper/component_commands.go

```diff
@@ -202,7 +202,7 @@ func (componentOps *gitComponent) getLatestTaggedFromBranch(repo, owner, branch,
 		return "", "", errors.Wrap(err, "Failed to get release tag refs from github client API")
 	}
 
-	branchCommits, _, err := componentOps.githubInterface.listCommits(owner, repo, &github.CommitsListOptions{SHA: branch})
+	branchCommits, _, err := componentOps.githubInterface.listCommits(owner, repo, &github.CommitsListOptions{ListOptions: github.ListOptions{PerPage: 100}, SHA: branch})
```

Comment:
While I'm OK with this, would it make sense to make this value configurable via a command line parameter ? Do users run this tool manually ?

Reply:
100 is the limit from github https://docs.github.com/en/rest/reference/commits#list-commits

This to is usually run by the a github action every x mintues

Reply:
"usually" is not exclusively .

Fair enough, if we don't bump manually, that's OK.